.travis.yml

```diff
@@ -12,6 +22,27 @@ install:
   - 'npm run vscode:prepublish'
   - 'npm install -g vsce'
   - 'vsce package'
+before_script:
+  # Download and provide in path kubectl , which is a requirement for using minikube.
+  - curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.17.4/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
```

Comment:
note: staying with 1.77.x for now due to 1.18.x having more requirements: https://github.com/lilic/travis-minikube/issues/24